appeals-service-api/src/__tests__/appeals.spec.js

```diff
@@ -8,7 +8,7 @@ const { appealDocument } = require('../models/appeal');
 jest.mock('../db/db');
 
 async function createAppeal() {
-  const appeal = appealDocument;
+  const appeal = JSON.parse(JSON.stringify(appealDocument));
```

Comment:
What's the story here? Surely this is the same as `appealDocument`

Reply:
I think we need a deep copy to avoid issues when tests etc are run in parallel.